### PR TITLE
Update managing user token perms docs

### PR DESCRIPTION
**Pull request by gibsondan**
- Viewers can create their own user tokens
- Only org admins can edit others user tokens

### CODE REVIEW

**Review by slopp — APPROVED** [review]
lgtm, we could add another line to this table about managing another uesr's token

https://github.com/dagster-io/dagster/pull/20128/files#diff-7abe0ee619949f2452486bc50abe6272aaac1e3fb66379dc58b9a094b2bf93f0R515-R517

File: docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx [hunk 1]
```diff
@@ -517,8 +517,8 @@ User tokens are accessed in the UI by navigating to **user menu (your icon) > Or
           user tokens
         </a>
       </td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">✅</td>
+      <td className="bg-red-50">✅</td>
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx:521]:
```suggestion
      <td className="bg-green-50">✅</td>
      <td className="bg-green-50">✅</td>
```

File: docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx [hunk 2]
```diff
@@ -528,15 +528,15 @@ User tokens are accessed in the UI by navigating to **user menu (your icon) > Or
       <td className="bg-red-50">❌</td>
       <td className="bg-red-50">❌</td>
       <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">❌</td>
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx:531]: This is a no-go here, right? If so:

```suggestion
      <td className="bg-red-50">❌</td>
```

File: docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx [hunk 3]
```diff
@@ -528,15 +528,15 @@ User tokens are accessed in the UI by navigating to **user menu (your icon) > Or
...
       <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>Revoke all user tokens</td>
       <td className="bg-red-50">❌</td>
       <td className="bg-red-50">❌</td>
       <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">❌</td>
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx:539]: Same here:

```suggestion
      <td className="bg-red-50">❌</td>
```

**Review by erinkcochran87 — COMMENTED** [review]
Did this change? If so, could you link the PR where the change was made?

**Review by erinkcochran87 — APPROVED** [review]

**Pull request merged**